README.md

```diff
@@ -12,21 +12,86 @@ The `lytix_diary` plugin provides a feature within Moodle where students can con
 
 ## Requirements
 
-- Moodle Version: 4.1+
-- PHP Version: 7.4+
+- Supported Moodle Version: 4.1 - 4.5
+- Supported PHP Version:    7.4 - 8.2
```

Comment:
do we test against PHP 8.3 as well in yml file? If yes, I would specify 7.4 - 8.3 here